Models/AdminModels/productModel.js

```diff
@@ -0,0 +1,88 @@
+import sequelize from '../../db';
+import CONSTANTS from '../../_CONSTANTS';
+import QUERIES from '../rawqueries';
+
+async function deleteProduct(productId) {
+  sequelize.query(`${QUERIES.deleteProductById}`, {
```

Comment:
`async` without `await`.

Reply:
Also, if you have a function that only runs a promise-based function, just return it and avoid labeling it as `async`.

Reply:
removed